api/views.py

```diff
@@ -199,6 +200,15 @@ class SpeciesList(generics.ListAPIView):
     queryset = Species.objects.all()
     serializer_class = SpeciesSerializer
 
+class GtoPIDList(generics.ListAPIView):
+
+    """
+    Get a list of Guide to Pharmacology ligand IDs
+    \n/species/
+    """
+
+    queryset = LigandID.objects.filter(web_resource_id=6).prefetch_related('ligand')
```

Comment:
The resource_id can change on build, better to use the slug of the resource instead.

Reply:
Good point. Corrected pointing to slug of guide to pharmacology ligand (gtoplig)